packages/core/src/lib/implementation/read-code-pushup-config.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { join } from 'path';
+import * as process from 'process';
+import { describe, expect } from 'vitest';
+import { persistConfig, uploadConfig } from '@code-pushup/models/testing';
+import { readCodePushupConfig } from './read-code-pushup-config';
+
+describe('readCodePushupConfig', () => {
+  it('should load file', async () => {
+    const module = await readCodePushupConfig(
+      join(process.cwd(), 'packages', 'core', 'test', 'minimal.config.mjs'),
+    );
+    expect(module).toEqual(
+      expect.objectContaining({
+        persist: expect.objectContaining(persistConfig()),
+        upload: expect.objectContaining({
+          ...uploadConfig(),
+          organization: 'code-pushup',
+        }),
+        categories: expect.arrayContaining([
+          expect.objectContaining({
+            slug: 'category-slug-1',
+          }),
+        ]),
+        plugins: expect.arrayContaining([
+          expect.objectContaining({
+            slug: 'mock-plugin-slug',
+          }),
+        ]),
+      }),
+    );
```

Comment:
The `expect.objectContaining` is useful when you only want check some object properties, but not all. Other than `categories` and `plugins`, it appears to me like you're matching everything anyway. Doesn't it also pass without the `expect.objectContaining`? 

```suggestion
    expect(module).toEqual({
      persist: persistConfig(),
      upload: {
        ...uploadConfig(),
        organization: 'code-pushup',
      },
      categories: expect.arrayContaining([
        expect.objectContaining({
          slug: 'category-slug-1',
        }),
      ]),
      plugins: expect.arrayContaining([
        expect.objectContaining({
          slug: 'mock-plugin-slug',
        }),
      ]),
    });
```

Reply:
no